documentation/tutorial.md

```diff
@@ -0,0 +1,70 @@
+# Charmed MongoDB tutorial:
+Ready to try out Charmed MongoDB? This tutorial will guide you through the steps to get Charmed MongoDB up and running. 
+
+## Minimum requirements:
+Before we start be sure you have the following requirements:
+- Ubuntu 20.04(focal) or later.
+- 8GB of RAM.
+- 2 CPU threads.
+- At least 20GB of available storage.
+- Access to the internet for downloading the required snaps and charms.
+
+## Install and prepare LXD:
+The fastest, simplest way to get started with Charmed MongoDB is to set up a local LXD cloud. LXD is a system container and virtual machine manager; Charmed MongoDB will be run in one of these containers and managed by Juju. While this tutorial covers the basics of LXD, you can explore more LXD [here](https://linuxcontainers.org/lxd/getting-started-cli/). The first step on our journey is to install LXD. LXD is installed from a snap package:
+```
+sudo snap install lxd --classic
+```
+
+Next we need to run `lxd init` to perform post-installation tasks. For this tutorial the default parameters are preferred and the network bridge should be set to have no ipv6 addresses:
+```
+lxd init --auto
+lxc network set lxdbr0 ipv6.address none 
+```
```

Comment:
I like this, but I think it would be best to exclude this for now. I remember Daniele saying a tutorial shouldn't give users multiple options to do something as it confuses readers on their journey